### PR TITLE
fix issue 223

### PR DESCRIPTION
**Pull request by londumas**
store the all 2d matrix of the do_cf1d()

### CODE REVIEW

**Comment by mblomqvist** [comment]
Looks good. If I can nitpick (before approving), I think that the comment on line 166 can be removed.

**Comment by londumas** [comment]
I like this comment, it explains à little bit what is done and acts as a paragraphe. Ok to merge ?

**Comment by mblomqvist** [comment]
In terms of style, the codes here typically don't have comments (it would be nice if they did, actually). You can keep it if you want, but please change the sentence into "To store the full 2D array" or something similar. I know, it's nitpicking...

**Comment by ngbusca** [comment]
I agree with @mblomqvist that this comment is not only not very informative but also quite cryptic! I'm for putting comments, but they should help understanding the code, warning people at key steps, etc. It appears as quite random here.

File: bin/do_cf1d.py [hunk 1]
```diff
@@ -162,6 +162,12 @@ def cf1d(p):
  
     w = wes>0
     cfs[w]/=wes[w]
+
+    ### To store the all 2d
+    cfs_2d = cfs.copy()
+    wes_2d = wes.copy()
+    nbs_2d = nbs.copy()
+
```
[the remark this answers is not in the document]
Comment (londumas) [reply bin/do_cf1d.py]: I have to copy these array, because after they are modified by the code.

**Comment by ngbusca** [comment]
Ok, I agree it's the easiest way. I would put a more explicit comment, e.g.

> ## make the copies of the arrays below that will be saved in the output file

Please also put here the plots of your tests.

**Comment by londumas** [comment]
the plots are here: https://github.com/igmhub/picca/issues/223
is it ok now ?

**Comment by ngbusca** [comment]
Thanks @londumas, if @mblomqvist agrees we can merge.

**Pull request merged**